utility/src/main/java/org/oppia/android/util/parser/TestGlideImageLoader.kt

```diff
@@ -1,18 +1,48 @@
 package org.oppia.android.util.parser
 
 import android.content.Context
+import android.graphics.Bitmap
 import android.graphics.drawable.Drawable
+import android.graphics.drawable.PictureDrawable
 import org.oppia.android.util.caching.AssetRepository
 import org.oppia.android.util.caching.CacheAssetsLocally
 import javax.inject.Inject
 
-/** A [TestGlideImageLoader] that is used in tests. */
+/** [TestGlideImageLoader] is designed to be used in tests. It uses real [GlideImageLoader]
+ * except for [loadDrawable]. [loadDrawable] function is overridden to work with drawable matchers
+ * in unit tests.
+ */
 class TestGlideImageLoader @Inject constructor(
   context: Context,
   @CacheAssetsLocally private val cacheAssetsLocally: Boolean,
   assetRepository: AssetRepository
-) : GlideImageLoader(context, cacheAssetsLocally, assetRepository) {
+) : ImageLoader {
 
+  private val glideImageLoader by lazy {
+    GlideImageLoader(context, cacheAssetsLocally, assetRepository)
+  }
+
+  override fun loadBitmap(
+    imageUrl: String,
+    target: ImageTarget<Bitmap>,
+    transformations: List<ImageTransformation>
+  ) {
+    glideImageLoader.loadBitmap(imageUrl, target, transformations)
+  }
+
+  override fun loadSvg(
+    imageUrl: String,
+    target: ImageTarget<PictureDrawable>,
+    transformations: List<ImageTransformation>
+  ) {
+    glideImageLoader.loadSvg(imageUrl, target, transformations)
+  }
```

Comment:
```suggestion
  override fun loadBitmap(
    imageUrl: String,
    target: ImageTarget<Bitmap>,
    transformations: List<ImageTransformation>
  ) = glideImageLoader.loadBitmap(imageUrl, target, transformations)

  override fun loadSvg(
    imageUrl: String,
    target: ImageTarget<PictureDrawable>,
    transformations: List<ImageTransformation>
  ) = lideImageLoader.loadSvg(imageUrl, target, transformations)
```

Optional Kotlin-specific way to override. This makes it a bit clearer that the overrides are trivial and single-purpose.

Reply:
Done, thanks!